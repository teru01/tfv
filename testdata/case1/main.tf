resource "my_resource" "myvar" {
  url        = "http://semvar.co.jp/${var.appname}/${var.env}"
  title      = var.moge
  expiration = var.expiration_days
}

locals {
  foo = <<EOF
    python
    ${var.limit} + ${var.requests}
    rust
  EOF
}

resource "my_server" "orange" {
  # title      = var.moge
  locations = var.locations

  /*
  apple {
    teste = "sweet"
    created_at = var.created_at
  }
  */

  mogura {
    meta = var.metadata
  }

  lang = var.lang
}
