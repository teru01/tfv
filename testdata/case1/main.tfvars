appname = "tfv"

unused = {
  "abc"  = "def"
  "bird" = "owl"
}

moge = ["world", "wide", "web"]
