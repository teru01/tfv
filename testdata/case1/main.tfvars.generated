appname = "tfv"
moge = ["world", "wide", "web"]