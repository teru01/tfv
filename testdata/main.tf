resource "aws_ecs_service" "mongo" {
  name            = "mongodb-${var.env}"
  cluster         = aws_ecs_cluster.foovar.id
  task_definition = aws_ecs_task_definition.mongo.arn
  desired_count   = 3
  iam_role        = var.iam_role_arn
  depends_on      = [aws_iam_role_policy.foo]

  load_balancer {
    target_group_arn = aws_lb_target_group.foo.arn
    container_name   = "mongo-${var.appname}"
    container_port   = 8080
  }

  placement_constraints {
    type       = "memberOf"
    expression = "attribute:ecs.availability-zone in [us-west-2a, us-west-2b]"
  }
}
