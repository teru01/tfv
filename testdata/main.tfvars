aaa = "hoge"

myobj = {
  "abc"  = "def"
  "bird" = "owl"
}
