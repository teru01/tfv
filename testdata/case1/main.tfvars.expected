env = "prd"

// 地名
locations = [
  "osaka",
  "aomori",
  # "kanagawa"
  "chiba"
]
